readme.md

```diff
@@ -198,7 +198,7 @@ If you want to have it for free with all the features, run the project yourself
     * Advanced features
     * Email reminders
     * Importers
-    * People who contribute to the GitHub repository (with a pull request that adds value, that gets merged (not a typo fix, for instance) will also have access to the Paid version for free.
+    * People who contribute to the GitHub repository (with a pull request that adds value, that gets merged--not a typo fix, for instance) will also have access to the Paid version for free.
```

Comment:
Thanks! It will be even better with an en dash '–' caracter.

Reply:
@asbiin: thanks for the feedback. I think that an 'em dash' would be [more appropriate here](https://www.chicagomanualofstyle.org/qanda/data/faq/topics/HyphensEnDashesEmDashes/faq0002.html), as en dashes are typically used to denote a range, i.e. January–September, as opposed to em dashes, which are typically used for sentence breaks.

**In any case, the first commit in this PR attempted to add an em dash (unicode 2014), but GitHub did not render the character. (See diff b54ec4d.) My takeaway was that GitHub doesn't want non-ASCII characters in the file, which is why I used two dashes.**

I changed the file back to an em dash in ecbf4cc; let me know if you can see the updated character. If not, we can look at a different solution. _This may be a bug in GitHub's online markdown editor._